main.cpp

```diff
@@ -13,75 +13,495 @@ What you create in this project will be used as the basis of Project 5 in the co
+        int memberId = 0;
+        //member's name
+        std::string name;
+        //member's role
+        std::string jobRole = "scientist";
+        //member's weight
+        float weight = 70.0f;
+        //member's age
+        float age =33.3f;
+        
+        //do main job
+        void examineAnimal (int date, float time, Cat cat);
+        //record results
+        bool recordTest (int date, float time, int testNum=0);
+        //do additional mandatory job
+        void examineCrew (int date, float time, crewMember memberId);
+    };
+
+    //dock to space station
+    bool dock();
+    //fly around planets
+    void makeLoop(int planetNum = 3);
+    //explore space
+    bool takeOf(float startTime);
+
+};
+
+/*
+Thing 5) DAC
+5 properties:
+    1) samplerate
+    2) bit depth
+    3) SNR
+    4) amount of power consumed
+    5) dynamic range
+3 things it can do:
+    1) recieve PCM signal
+    2) correct errors 
+    3) produce analogue signal
+ */
+struct DAC
+{
+    //samplerate
+    int sampleRate = 48000;
+    //bit depth
+    int bitDepth = 24;
+    //SNR
+    int SNR = 112;
+    //amount of power consumed
+    float amountOfPoerwConsumed = 0.9f;
+    //dynamic range
+    double dynamicRange = 9.9;
+    
+    //recieve PCM signal
+    void readInput(int channelNum=0);
+    //correct errors 
+    bool checkError(long double sampleNum);
+    //produce analogue signal
+    void audioOut(int channelNum=0);
+};
+/*
+Thing 6) power unit
+5 properties:
+    1) weight
+    2) otput voltage
+    3) input voltage
+    4) output current
+    5) operating and storage temperature ranges
+3 things it can do:
+    1) input power from outlet
+    2) convert voltage
+    3) protect circuit
+ */
+
+struct PowerUnit
+{
+    //weight
+    float weight = 1.2f;
+    //otput voltage
+    double outVolt = 12;
+    //input voltage
+    double inVolt = 220;
+    //output current
+    int outCurrent = 1;
+    //operating and storage temperature ranges
+    int maxOutTemp = 80;
+     
+    //input power from outlet
+    bool getElectricity(int outletStandart=1);
+    //convert voltage
+    double convertVoltage(double inVolt, double outVolt);
+    //protect circuit
+    bool status(int circuitId=1);
+};
+/*
+Thing 7) VCA
+5 properties:
+    1) frequency responce
+    2) insertion loss
+    3) attenuation
+    4) price
+    5) number of channels
+3 things it can do:
+    1) attenuate signal
+    2) consume power
+    3) recieve control signal
+ */
+struct VCA
+{
+    //frequency responce
+    float freqResponse = 0.5f;
+    //insertion loss
+    double insertLoss = 0.01;
+    //attenuation
+    int attenuation = -100;
+    //price
+    float price = 3;
+    //number of channels
+    int channelNum = 2;
+    
+    //attenuate signal
+    void attenuate(int coefficient = -50);
+    //consume power
+    void inputPower(int amountOfPower = 20);
+    //recieve control signal
+    int readKnob (long double knobAngle = 10.123);
```

Comment:
you're initializing a `long double` with a `double`, causing a warning.  change your `long double` to `double` 
- [ ] revise